### ISSUE TITLE
optionally tag a team or contact person if a PR is merged with negative effects?

### ISSUE THREAD

**Issue by mr-c** [hauntsaninja/mypy_primer#128]
Example, in https://github.com/python/mypy/pull/17724#issuecomment-2320985436 and other it would have been nice to have been `@` tagged in a comment after that PR was merged as a heads-up.

(In this case, specifically for `cwltool`; it isn't a big deal, but fewer surprises are better!)

**Comment by hauntsaninja** [comment]
Yeah, happy to ping you. Don't think I was involved in either the typeshed PR or the sync to mypy though

Implementing this feature programmatically might be a little tricky, ideas/contributions welcome

**Comment by jorenham** [comment]
Maybe display the gh username of the person to contact next to the repo name (without actually tagging to avoid spam tsunamis)?
